3_javascript/javascript1/week2/homework javascript1 week2/candy.js

```diff
@@ -0,0 +1,49 @@
+//Candy helper optional //
+
+const boughtCandyPrices = [];
+
+const amountToSpend = Math.random() * 100;
+
+const candyPricePalette = [
+  {
+    type: "sweet",
+    price: 0.5,
+  },
+  {
+    type: "chocolate",
+    price: 0.07,
+  },
+  {
+    type: "toffee",
+    price: 1.1,
+  },
+  {
+    type: "chewing-gum",
+    price: 0.2,
+  },
+];
+function addCandy(candyType, weight) {
+  for (let i = 0; i < candyPricePalette.length; i++) {
+    if (candyType === candyPricePalette[i].type) {
+      boughtCandyPrices.push(candyPricePalette[i].price * weight);
+      break;
+    }
+  }
+}
+function canBuyMoreCandy() {
```

Comment:
Look good and nice use of the addition-assignment operator (`+=`)! 😋 I assume initializing `sum` to `60` is a typo?

Reply:
yes, it was a typo.